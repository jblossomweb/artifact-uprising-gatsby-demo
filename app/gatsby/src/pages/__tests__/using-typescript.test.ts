import snapshotTests from "../../../utils/story-snapshot-tests"

jest.mock("../../layouts/default")
jest.mock("../../components/seo")

import { stories } from "./using-typescript.stories"

describe("pages/using-typescript", () => {
  snapshotTests(stories)
})
