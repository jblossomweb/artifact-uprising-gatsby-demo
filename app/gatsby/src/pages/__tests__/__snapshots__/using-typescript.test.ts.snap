// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/using-typescript build time matches snapshot 1`] = `
Array [
  <div
    className="sc-AxirZ eWvgOi"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-1 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <a
              href="/"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                />
              </svg>
            </a>
          </span>
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
        >
          Test Title
        </h6>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Cart 
            <span
              className="MuiBadge-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
              <span
                className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
              >
                0
              </span>
            </span>
          </span>
        </button>
      </div>
    </header>
  </div>,
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <main
      style={
        Object {
          "marginTop": "72px",
        }
      }
    >
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /using-typescript/
        " which was built on
         
        2020-07-10 02:22 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
  </div>,
  <footer />,
]
`;

exports[`pages/using-typescript path matches snapshot 1`] = `
Array [
  <div
    className="sc-AxirZ eWvgOi"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-1 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <a
              href="/"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                />
              </svg>
            </a>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
        >
          Test Title
        </h6>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Cart 
            <span
              className="MuiBadge-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
              <span
                className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
              >
                0
              </span>
            </span>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
  </div>,
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <main
      style={
        Object {
          "marginTop": "72px",
        }
      }
    >
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /a-different-path
        " which was built on
         
        2020-07-11 03:26 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
  </div>,
  <footer />,
]
`;

exports[`pages/using-typescript sample matches snapshot 1`] = `
Array [
  <div
    className="sc-AxirZ eWvgOi"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-1 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <a
              href="/"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                />
              </svg>
            </a>
          </span>
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
        >
          Test Title
        </h6>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Cart 
            <span
              className="MuiBadge-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
              <span
                className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
              >
                0
              </span>
            </span>
          </span>
        </button>
      </div>
    </header>
  </div>,
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <main
      style={
        Object {
          "marginTop": "72px",
        }
      }
    >
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /using-typescript/
        " which was built on
         
        2020-07-11 03:26 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
  </div>,
  <footer />,
]
`;
